raiden/tests/unit/test_startup.py

```diff
@@ -0,0 +1,156 @@
+from copy import deepcopy
+from typing import Any, Dict
+
+import pytest
+
+from raiden.app import App
+from raiden.constants import Environment
+from raiden.ui.startup import setup_contracts_or_exit, setup_environment, setup_network_id_or_exit
+from raiden_contracts.constants import (
+    CONTRACT_ENDPOINT_REGISTRY,
+    CONTRACT_SECRET_REGISTRY,
+    CONTRACT_SERVICE_REGISTRY,
+    CONTRACT_TOKEN_NETWORK_REGISTRY,
+    CONTRACT_USER_DEPOSIT,
+)
+
+
+class MockWeb3Version():
+
+    def __init__(self, netid):
+        self.network = netid
+
+
+class MockWeb3():
+
+    def __init__(self, netid):
+        self.version = MockWeb3Version(netid)
+
+
+def test_setup_network_id():
+    config = {}
+
+    # Normal test
+    netid, known = setup_network_id_or_exit(config, 68, MockWeb3(68))
+    assert netid == 68
+    assert not known
+    assert config['chain_id'] == netid
+
+    # Chain id different than the one in the ethereum client
+    with pytest.raises(SystemExit):
+        setup_network_id_or_exit(config, 61, MockWeb3(68))
+
+    # Known network ids
+    for netid in (1, 3, 4, 42, 627):
```

Comment:
With these tests you can remove the one in the beginning.

Reply:
No I can't. This is not really obvious though due to a lack of a proper comment. I will add it.

So the "Normal test" is of a random unknown (private chain) network id. The ones in the range are all known networks. Mainnet, ropsten, rinkeby, kovan, smoketest.

Reply:
What about a parametrized test then to make this clearer?